### PR TITLE
Add validation to check argument

### PR DESCRIPTION
**Pull request by KotaTakahashi9320**
# Summary
Add validation to check argument
# Purpose
To prevent requests with abnormal values
# Contents
Add size check
Add data format check
Add test
# Testing Methods Performed
`pipenv run pytest`

### CODE REVIEW

File: tests/unit_tests/test_qmpc.py [hunk 1]
```diff
@@ -14,14 +14,47 @@ class TestQMPC:
...
+        with pytest.raises(Exception):
+            # 行が足りずシェアがない
+            self.qmpc.send_share(
+                [], ["attr1", "attr2", "attr3"], 1, 1000)
+        with pytest.raises(Exception):
+            # schemaに同じものが含まれる
+            self.qmpc.send_share(
+                [[1, 1, 2], [2, 3, 4]], ["id", "a", "a"], 1, 1000)
+        with pytest.raises(Exception):
+            self.qmpc.send_share(
```
Comment (mdonaka) [inline tests/unit_tests/test_qmpc.py:57]: Please add a comment since it is missing.

File: tests/unit_tests/test_qmpc.py [hunk 2]
```diff
@@ -14,14 +14,47 @@ class TestQMPC:
...
+                         run_server1, run_server2, run_server3):
         """ serverにシェアを送れるかのTest"""
         response: Dict[str, Any] = self.qmpc.send_share(
-            secrets, [], 1, 1000)
+            secrets, schema, 1, 1000)
         assert (response["is_ok"])
 
+    def test_send_shares_errorhandring(self,
+                                       run_server1, run_server2, run_server3):
+        with pytest.raises(Exception):
```
Comment (mdonaka) [inline tests/unit_tests/test_qmpc.py]: I would like `Exception` to be more accurate.
For example, `RuntimeError` and `ArgmentError`....

**Review by mdonaka — COMMENTED** [review]
@KotaTakahashi9320 
Why is `proto/*` updated?

**Comment by KotaTakahashi9320** [comment]
@mdonaka
Thanks for the review!
> Why is `proto/*` updated?

It was corrected when I execute `pipenv run make fmt`
I fixed it with [0464603](https://github.com/acompany-develop/QuickMPC-libClient-py/pull/18/commits/046460366a53b1b0d77056f22d3b7b304e2aee45)

Please check again

**Comment by mdonaka** [comment]
@KotaTakahashi9320 
> It was corrected when I execute pipenv run make fmt
I fixed it with [0464603](https://github.com/acompany-develop/QuickMPC-libClient-py/pull/18/commits/046460366a53b1b0d77056f22d3b7b304e2aee45)

I see. I guess I should either include `proto/*` in flake8 or exclude it from format...
Either way, thanks for this change!

File: tests/unit_tests/test_qmpc.py [hunk 3]
```diff
@@ -14,14 +15,49 @@ class TestQMPC:
...
+        with pytest.raises(TypeError):
+            # シェアがない
+            self.qmpc.send_share(
+                [], ["attr1", "attr2", "attr3"], 1, 1000)
+        with pytest.raises(RuntimeError):
+            # schemaに同じものが含まれる
+            self.qmpc.send_share(
+                [[1, 1, 2], [2, 3, 4]], ["attr1", "attr2", "attr2"], 1, 1000)
+        with pytest.raises(RuntimeError):
+            # 正方行列でない
```
Comment (mdonaka) [inline tests/unit_tests/test_qmpc.py]: Perhaps "行によって列数が異なる" or something similar would be appropriate.
For example, `[[1, 1, 2], [2, 3, 4]]` is not "正方行列", but OK.

Comment (KotaTakahashi9320) [reply tests/unit_tests/test_qmpc.py]: That's right! Thanks.
I fixed it with [2838ba6](https://github.com/acompany-develop/QuickMPC-libClient-py/pull/18/commits/2838ba663ebe93ed049362d8a4bcbd877e62795e)

**Review by mdonaka — APPROVED** [review]

**Review by arukuka — APPROVED** [review]
Thank you!

**Pull request merged**